linode/manager
# M3-5168: Fix "Unknown Plan" and "Unknown Image" in Search

## Description

- Fixes `Unknown Image` and `Unknown Type` in Cloud Manager's search.
  - Fixes plan names and image names showing for large customers
  - Fixed `Unknown Image` for LKE Linodes

Before(s):
<img width="1472" alt="Screen Shot 2021-07-27 at 4 21 29 PM" src="https://user-images.githubusercontent.com/6440455/127222286-ab2309a9-6d46-4b3b-8b5b-00b6ac93c007.png">
<img width="1472" alt="Screen Shot 2021-07-27 at 4 22 06 PM" src="https://user-images.githubusercontent.com/6440455/127222378-703ee698-3685-4424-9cb3-23f5d906e428.png">
<img width="857" alt="Screen Shot 2021-07-27 at 4 31 02 PM" src="https://user-images.githubusercontent.com/6440455/127223518-d1aae61e-dfc8-43cf-9eaa-d7c403c4d566.png">

After(s):

<img width="1472" alt="Screen Shot 2021-07-27 at 4 21 46 PM" src="https://user-images.githubusercontent.com/6440455/127222321-eeb27295-e9c7-40d8-bda7-fd851e15ae03.png">
<img width="1472" alt="Screen Shot 2021-07-27 at 4 21 57 PM" src="https://user-images.githubusercontent.com/6440455/127222353-2bd6a022-6697-4138-8bff-225663da319c.png">

## How to test

- Test searching on accounts
  - Test on Large and Small accounts
  - Try searching for different types of entities
  - Ensure no functionality has been lost as compared to `cloud.linode.com`
